### DISCUSSION TITLE
Some questions about public transport routes

### DISCUSSION THREAD

**Discussion by Lewatoto** [osmandapp/OsmAnd#14176] [Maps Related]
Hello, today I subscribed to the monthly plan to work on some public transport routes, after mapping the following doubts arose:

### The info of bus stops and routes is updated in the hourly update?
On this bus stop, already change the name at the beginning of the last month and didn't update on the map.
**Old name showed on OsmAnd**
![image](https://user-images.githubusercontent.com/8757169/166181850-436873ee-7ced-4d8c-9207-ffa57765287d.png)
**Updated name showed on osm data**
![image](https://user-images.githubusercontent.com/8757169/166181893-a373aa32-4720-4184-9598-d37cbb5086b2.png)
Link to the bus stop/station https://www.openstreetmap.org/node/4201162890

Even on the route didn't show some stops, all stops are listed here https://www.openstreetmap.org/relation/6245758#map=15/14.6412/-90.4955&layers=O
![image](https://user-images.githubusercontent.com/8757169/166190112-37563bd3-02eb-46b2-bd7d-0f98e9db95ad.png)

### Bus stop render using PTV2 relation does not show the square with the color (or show different color) and name of routes that use that stop
I'm working on changing all the BRT routes of my city to PTV2 following https://wiki.openstreetmap.org/wiki/Public_transport#Service_routes and recently I've found https://osmand.net/blog/guideline-pt, the platforms that are part of each route does not show the info of the route or route master.

This stop belong to route **Transmetro L6: Plaza Barrios-FEGUA -> Proyectos** witch have as master relation **Transmetro L6** but that info isn't showed meanwhile the route of the first image in which I didn't follow either version of PT displays the data.
![image](https://user-images.githubusercontent.com/8757169/166183753-a8d68687-0b58-4a51-a418-95eeff3ae020.png)
The only way to see the route info is navigating and use the public transport but the color of the route shows different, it show some brown color and the color that have on osm is #ffdd00.
![image](https://user-images.githubusercontent.com/8757169/166183921-17580d60-cbdc-4551-9d96-eb36208f0a93.png)
Link to route master L6 https://www.openstreetmap.org/relation/14091642

By the way, for what it's worth, I'm using the osm web editor and not josm because the latter looks too small on my pc and if I use the interface scaling option, the entire interface becomes too big and therefore useless.

Maybe I mess with some tags of PTV2 already known that I've only add platforms and not stop positions, any help is appreciated.

**Comment by vshcherb** [comment]
There are issues with Public Transport updates in OsmAnd Live. Unfortunately mostly edit of existing routes doesn't work, there are open issues on Github https://github.com/osmandapp/OsmAnd/issues/10570 https://github.com/osmandapp/OsmAnd/issues/6743

**Reply by Lewatoto** [reply]
ok, I understand and the mismatched color is also problem of route or since is to bright OsmAnd changed to be more readable? 

As side note you should add a warning on the subscription menu that says the routes and the items related have bugs and it's possible that didn't update even if you buy the subscription that have the hourly update, just to avoid bad reviews, legal issues or things like that.

**Reply by vshcherb** [reply]
Yes, everything that is marked as OSM relation could be a problem. They sometimes disappear after updates, so the only working 100% update is to add new route. Updating existing relations could lead to the flaws.
